app/404.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Page Not Found :(</title>
+    <style>
```

Comment:
I know this is an early placeholder, but we should always use CSS links, and not the inline version.